### PR TITLE
bump network addons version to 0.4.0

### PR DESCRIPTION
**Pull request by phoracek**
I forgot to generate new manifests in #25. Fixes https://github.com/kubevirt/hyperconverged-cluster-operator/issues/27.

Also bumps version of network addons operator.

### CODE REVIEW

**Comment by phoracek** [comment]
@tripledes @rthallisey @djzager could you please take a look?

**Comment by phoracek** [comment]
/hold

Generated manifests are invalid. e.g. image path leads to docker.io, while we host it on quay.io

**Comment by phoracek** [comment]
Travis has some issues, waiting for builds https://travis-ci.com/kubevirt/cluster-network-addons-operator/branches to finish, so I can verify this PR.

File: hack/defaults [hunk 1]
```diff
@@ -23,6 +23,12 @@ APISERVER_IMAGE="${APISERVER_IMAGE:-cdi-apiserver}"
 UPLOADPROXY_IMAGE="${UPLOADPROXY_IMAGE:-cdi-uploadproxy}"
 UPLOADSERVER_IMAGE="${UPLOADSERVER_IMAGE:-cdi-uploadserver}"
 
+NETWORK_ADDONS_MULTUS_IMAGE="${NETWORK_ADDONS_MULTUS_IMAGE:-docker.io/nfvpe/multus:latest}"
```
Comment (phoracek) [inline hack/defaults]: @myakove this should help with d/s testing

**Comment by phoracek** [comment]
/hold cancel

operator-test.sh passed. @rthallisey @djzager could you please review? Thanks

File: hack/defaults (hunk 1 above)
Comment (myakove) [reply hack/defaults]: Look good, @lukas-bednar is this ok?

Comment (lukas-bednar) [reply hack/defaults]: Yes I will set these params accordingly. If all images are ready I can even test it today.

**Pull request merged**